### DISCUSSION TITLE
Suggestion

### DISCUSSION THREAD

**Discussion by Jeiel0rbit** [iamrasel/lunar-launcher#26] [Ideas]
~~Hi! The app is very good. Able to save battery and consume less resources. I suggest customizing the navigations in the applications. For example: browse horizontally. Navigate with dual columns. Keeping the words. Try customizing the volume buttons, although this can consume battery, cpu and ram.~~

**Comment by iamrasel** [comment]
@Jeiel17 Sorry, I couldn't understand what you said. By mentioning `navigations in the applications` and `volume buttons`, exactly which sections you tried to indicate? Better explanations with proper examples would be helpful.

**Reply by Jeiel0rbit** [reply]
Just customization. The cross-application denial is vertical. Put it horizontally too.

**Reply by iamrasel** [reply]
If I get it right, I think you are talking about landscape screen support. Now I can't work on those, maybe in future.

**Comment by Terkyz** [comment]
In some applications like Stealth or Infinity when opening a link within the app it opens the default browser used by the user but it is only a [session](https://i.imgur.com/qeJ10Bj.jpg) to view that specific link, it does not create a tab in the user's browser, wouldn't it be better if Lunar RSS worked like this?

**Reply by iamrasel** [reply]
I think you're saying about custom tabs, can be done.

**Comment by Terkyz** [comment]
I had another idea, maybe it would be useful an option to disable the effects that the launcher has? Effects like when the names of the applications appear when searching in the app drawer.

**Reply by iamrasel** [reply]
Those are android's built-in animations, you can disable them from your android's settings.

**Reply by Terkyz** [reply]
> Those are android's built-in animations, you can disable them from your android's settings.

You are right, maybe it would be a good idea to add that detail to the FAQ or to the Lunar Wiki.
